jina/peapods/zmq/__init__.py

```diff
@@ -495,7 +507,11 @@ def send_message(
     num_bytes = 0
     try:
         _prep_send_socket(sock, timeout)
-        sock.send_multipart(msg.dump())
+        if 'flags' in kwargs:
+            kwargs['flags'] |= zmq.NOBLOCK
+        else:
+            kwargs['flags'] = zmq.NOBLOCK
+        sock.send_multipart(msg.dump(), **kwargs)
```

Comment:
What else could be in `kwargs`? It was not used before, so why use it now?

Reply:
It contained the default flags. We needed to add this specific one